coremltools/converters/mil/frontend/torch/torchir_passes.py

```diff
@@ -70,13 +70,41 @@ def forward(self, x):    # x a tensor with shape [4,10]
         def forward(self, x):    # x a tensor with shape [5, 4]
             x[2] = 9
             return x
+
+    In case:
+
+        def forward(self, x):    # x a tensor with shape [4,10]
+            y = torch.empty(*x.shape)
+            y.copy_(0)
+            return y
+
+    Input graph:
+
+        input -> %x
+        %y = empty[](x.shape)
+        %1 = copy_[](%y, %x)
+        return (%1)
+        output -> %1
+
+    In result of fuse
+
+        input -> %x
+        %y = [empty[](x.shape)]
+        %x_internal_tensor_assign_1 = _internal_op_tensor_inplace_copy(%y, %x)
+        output -> %x_internal_tensor_assign_1
```

Comment:
My comment may be insufficient.
What I want to say is graph of torch.jit from `x[1:2, 1:2] = [[0]]; return x` and `x.copy_(); return x`is different.
In latter case, output of `copy_` node will be used in following graph.